modules/apps/translation/translation-web/src/main/resources/META-INF/resources/import_translation_results.jsp

```diff
@@ -54,121 +54,163 @@ renderResponse.setTitle(LanguageUtil.get(resourceBundle, "import-translation"));
 
 		<clay:container-fluid
 			cssClass="container-view"
+			size="lg"
 		>
-			<clay:sheet
-				cssClass="translation-import-body-form"
-			>
-				<h3 class="mb-4"><%= importTranslationResultsDisplayContext.getFileName() %></h3>
-
-				<c:if test="<%= importTranslationResultsDisplayContext.getSuccessMessagesCount() > 0 %>">
-					<h4 class="text-success">
-						<span class="mr-2">
-							<clay:icon
-								symbol="check-circle-full"
-							/>
-						</span>
-
-						<%= importTranslationResultsDisplayContext.getSuccessMessageLabel(locale) %>
-					</h4>
+			<div class="translation-import-body-form">
 
-					<ul class="list-group list-group-no-bordered list-group-sm">
+				<%
+				boolean importTranslationResultsErrors = false;
 
-						<%
-						for (String successMessage : importTranslationResultsDisplayContext.getSuccessMessages()) {
-						%>
+				if (importTranslationResultsDisplayContext.getFailureMessagesCount() > 0) {
+					importTranslationResultsErrors = true;
+				}
+				%>
 
-							<li class="align-items-center list-group-item list-group-item-flex">
-								<div class="autofit-col autofit-col-expand">
-									<section class="autofit-section">
-										<div class="list-group-title"><%= successMessage %></div>
-									</section>
-								</div>
-
-								<div class="autofit-col text-right text-success">
-									<div class="list-group-icon">
-										<clay:icon
-											symbol="check-circle-full"
-										/>
+				<c:if test="<%= importTranslationResultsDisplayContext.getSuccessMessagesCount() > 0 %>">
+					<div>
+						<div class="panel panel-secondary" role="tablist">
+							<button aria-expanded="false" class="<%= importTranslationResultsErrors ? "collapsed" : StringPool.BLANK %> btn btn-unstyled collapse-icon collapse-icon-middle panel-header panel-header-link" role="tab" type="button">
```

Comment:
Are we controlling this aria-expanded someway?

Reply:
Sadly not, I forget it but I'm going to add it.